build/release/README.md

```diff
@@ -79,6 +79,12 @@ After the PR is merged, you can tag the release with the beta tag (`v1.13.0-beta
 
 **IMPORTANT** Before tagging the release, open a new PR to update the documentation and example manifest tags to the release version.
 
+The script  `set-release-ver.sh` can help preparing these changes:
```

Comment:
Have you looked into enabling whitespace visibility in your editor yet to help prevent double space issues? Please do this.

```suggestion
The script `set-release-ver.sh` can help preparing these changes:
```

Reply:
@BlaineEXE wrote:
> Have you looked into enabling whitespace visibility in your editor yet to help prevent double space issues?

yes but no conclusion yet.


> Please do this.

I will continue and find a solution